nipoppy/data/examples/sample_bids_dataset_description.json

```
@@ -0,0 +1,5 @@
+{
+    "Name": "CHANGEME",
+    "BIDSVersion": "1.10.1",
+    "DatasetType": "study"
```

I think it would be great to provide more in the template to guide people.

```suggestion
    "DatasetType": "study",
    "Authors": ["TODO"],
    "Description": "TODO: CHANGEME.  This an other fields are described at https://bids-specification.readthedocs.io/en/stable/modality-agnostic-files/dataset-description.html",
    "Funding": "TODO",
    "GeneratedBy": [{
        "Name": "nipoppy",
        "CodeURL": "https://github.com/nipoppy/nipoppy",
        "Version": "HOWTOINSERT nipoppy version?"
    }]
```